WooCommerce/src/androidTest/kotlin/com/woocommerce/android/e2e/screens/products/ProductListScreen.kt

```diff
@@ -57,7 +89,17 @@ class ProductListScreen : Screen {
                 ViewMatchers.withChild(
                     Matchers.allOf(
                         ViewMatchers.withId(R.id.productStockAndStatus),
-                        ViewMatchers.withText(product.stockStatus + " • \$${product.priceDiscountedRaw}.00")
+                        ViewMatchers.withText(
+                            product.stockStatus + product.variations + " • \$${product.priceDiscountedRaw}.00"
```

Comment:
what do you think about using only string interpolation instead of mixing string interpolation with the "+" operator?

```suggestion
                            "${product.stockStatus}${product.variations} • \$${product.priceDiscountedRaw}.00"

```

Reply:
It might be a matter of preference, in this particular case both end up taking the same amount of code, and subjectively, the non-interpolated option is easier to parse visually. I don't have a strong opinion here, so I went with your suggestion.